SharpMUSH.Client/Pages/WikiPageDiff.razor.css

```
@@ -0,0 +1,210 @@
+    padding: 0.125rem 0;
+    color: var(--text-faint);
+}
+.wikidiff-gutter--add { color: var(--accent); }
+.wikidiff-gutter--del { color: #e57373; }
+.wikidiff-line--ctx .wikidiff-text { color: var(--text-dim); }
+.wikidiff-line--del .wikidiff-text { color: var(--text-dim); }
+.wikidiff-text {
+    padding: 0.125rem 0.875rem;
+    color: var(--text);
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)